src/NServiceBus.Core/Features/SatelliteDefinitions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NServiceBus
+{
+    using System.Collections.Generic;
+
+    class SatelliteDefinitions
+
+    {
+        public void Add(SatelliteDefinition satelliteDefinition)
+        {
+            satelliteDefinitions.Add(satelliteDefinition);
+        }
+
+        public IEnumerable<SatelliteDefinition> Definitions => satelliteDefinitions;
```

Comment:
why having this IEnumerable property? Looks like we could just expose the list directly since it's internal?